docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -1062,8 +1062,8 @@ end::stats[]
 
 tag::stored_fields[]
 `stored_fields`::
-(Optional, Boolean) If `true`, retrieves the document fields stored in the
-index rather than the document `_source`. Defaults to `false`.
+(Optional, string)
+A comma-separated list of <<mapping-store,`stored`>> fields to include in the response.
```

Comment:
I included a link to the `mapping-store` docs. I noticed some other parameter descriptions did similar.

Reply:
I have a minor preference to have the URL refer `stored fields` instead of just `stored`.